hello.py

```diff
@@ -1 +1,10 @@
-print("Hello, world!")
+def hello() -> None:
+    '''
+    Prints out hello word to stdout.
```

Comment:
I prefer imperatively-worded docstrings (like a command):

 ```suggestion
    """Print "Hello, world!"."""
```